### PR TITLE
Ctor 619 refresh applications pfsense fauxapi

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 1]
```diff
@@ -5,62 +5,76 @@ title: Pfsense Fauxapi
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-
 ## Vue d'ensemble
 
 Dans son principe, FauxAPI lit simplement le fichier config.xml de pfSense, le convertit en JSON et retourne à l'appelant de l'API.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:10]: le?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 2]
```diff
@@ -77,113 +91,349 @@ Au niveau des droits API, votre fichier de configuration doit contenir au minimu
...
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
 
 ```bash
-yum install centreon-plugin-Applications-Pfsense-Fauxapi.noarch
+dnf install centreon-pack-applications-pfsense-fauxapi
 ```
 
-2. Sur l'interface Web de Centreon, installer le connecteur de supervision *Pfsense Fauxapi* depuis la page **Configuration > Gestionnaire de connecteurs de supervision**
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:119]:
```suggestion
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 3]
```diff
@@ -77,113 +91,349 @@ Au niveau des droits API, votre fichier de configuration doit contenir au minimu
...
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
 
 ```bash
-yum install centreon-pack-applications-pfsense-fauxapi.noarch
+dnf install centreon-plugin-Applications-Pfsense-Fauxapi
 ```
 
-3. Sur l'interface Web de Centreon, installer le connecteur de supervision *Pfsense Fauxapi* depuis la page **Configuration > Gestionnaire de connecteurs de supervision**
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:164]:
```suggestion
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
```

File: pp/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 4]
```diff
@@ -5,60 +5,74 @@ title: Pfsense Fauxapi
...
-More information about collected metrics is available in the official Pfsense Fauxapi documentation: https://github.com/ndejong/pfsense_fauxapi
+The connector brings the following service templates (sorted by the host template they are attached to):
 
 <Tabs groupId="sync">
-<TabItem value="Backup-files" label="Backup-files">
+<TabItem value="App-Pfsense-Fauxapi-custom" label="App-Pfsense-Fauxapi-custom">
+
+| Service Alias | Service Template                        | Service Description                          |
+|:--------------|:----------------------------------------|:---------------------------------------------|
+| Backup-Files  | App-Pfsense-Backup-Files-Fauxapi-custom | Check the Backup files Pfsense using Fauxapi |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:28]:
```suggestion
| Backup-Files  | App-Pfsense-Backup-Files-Fauxapi-custom | Check the Pfsense Backup files using Fauxapi |
```

File: pp/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 5]
```diff
@@ -5,60 +5,74 @@ title: Pfsense Fauxapi
...
+| Gateways      | App-Pfsense-Gateways-Fauxapi-custom     | Check the Gateways Pfsense using Fauxapi     |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:29]:
```suggestion
| Gateways      | App-Pfsense-Gateways-Fauxapi-custom     | Check the Pfsense Gateways using Fauxapi     |
```

File: pp/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 6]
```diff
@@ -5,60 +5,74 @@ title: Pfsense Fauxapi
...
+| Rules         | App-Pfsense-Rules-Fauxapi-custom        | Check the rules Pfsense using Fauxapi        |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:30]:
```suggestion
| Rules         | App-Pfsense-Rules-Fauxapi-custom        | Check the Pfsense rules using Fauxapi        |
```

File: pp/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 7]
```diff
@@ -74,113 +88,349 @@ permit = config_backup_list, gateway_status, rule_get, system_stats
...
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
 
 ```bash
-yum install centreon-plugin-Applications-Pfsense-Fauxapi.noarch
+dnf install centreon-pack-applications-pfsense-fauxapi
 ```
 
-2. On the Centreon Web interface in **Configuration > Monitoring Connector Manager**, install the *Pfsense Fauxapi* Monitoring Connector
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:116]:
```suggestion
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 8]
```diff
@@ -5,60 +5,74 @@ title: Pfsense Fauxapi
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-
 ## Overview
 
-At its core FauxAPI simply reads the core pfSense config.xml file, converts it to JSON and returns to the API caller. 
-FauxAPI provides easy backup and restore API interfaces that by default store configuration backups on all configuration write operations thus it is very easy to roll-back
-even if the API user manages to deploy a “very broken” configuration.
+At its core FauxAPI simply reads the core pfSense config.xml file, converts it to JSON and returns to the API caller.
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:10]:
```suggestion
```

File: pp/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 9]
```diff
@@ -5,60 +5,74 @@ title: Pfsense Fauxapi
...
+FauxAPI provides easy backup and restore API interfaces that by default store configuration backups on all configuration write operations thus it is very easy to roll-back even if the API user manages to deploy a "very broken" configuration.
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:11]:
```suggestion
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md (hunk 1 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:10]:
```suggestion
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 10]
```diff
@@ -5,62 +5,76 @@ title: Pfsense Fauxapi
...
 
-
 ## Vue d'ensemble
 
 Dans son principe, FauxAPI lit simplement le fichier config.xml de pfSense, le convertit en JSON et retourne à l'appelant de l'API.
-FauxAPI fournit des interfaces API de sauvegarde et de restauration faciles qui, par défaut, stockent les sauvegardes de la configuration 
-sur toutes les opérations d'écriture de la configuration ; il est donc très facile de revenir en arrière même si l'utilisateur de l'API parvient à déployer 
-une configuration "très cassée".
+FauxAPI fournit des interfaces API de sauvegarde et de restauration faciles qui, par défaut, stockent les sauvegardes de la configuration
+sur toutes les opérations d'écriture de la configuration ; il est donc très facile de revenir en arrière même si l'utilisateur de l'API parvient à déployer une configuration "très cassée".
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:12]:
```suggestion
```

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**